packages/charts-bar/src/charts-bar.scss

```diff
@@ -0,0 +1,61 @@
+// scss-lint:disable all
+
+.ibm-chart {
+
+  @import "https://raw.githubusercontent.com/IBM-Design/colors/master/_ibm-colors.scss";
```

Comment:
Before v1.0.0, we should reference the colors package in this project. We should add it as a peer dependency.